Automattic/wp-calypso
# My Home: Tweak styles for experimental layout

#### Changes proposed in this Pull Request

Tweak some styles and add alternative versions of some cards for better integration in the experimental Home layout, so the styling is more aligned with the mockups shared in pbAPfg-g0-p2.

Before | After
--- | ---
![image](https://user-images.githubusercontent.com/1233880/79974360-7e2d9280-8499-11ea-8f2b-939e2647fddf.png) | ![image](https://user-images.githubusercontent.com/1233880/79974369-85ed3700-8499-11ea-8d80-d42e570c1f4c.png)
![image](https://user-images.githubusercontent.com/1233880/79974386-8c7bae80-8499-11ea-8b97-8730493c9dba.png) | ![image](https://user-images.githubusercontent.com/1233880/79974394-900f3580-8499-11ea-9c3d-5af0131dfda8.png)

- New compact view for the quick links.
- Educational content is stacked (one card below the other) in a shared section.
- Font sizes and spacing have been adjusted.
- Display the "Connect accounts" task if there are no accounts connected.

If this PR feels too big, I'm happy to split work in different PRs.

#### Testing instructions

- Visit My Home without enabling the experimental layout: `/home/:site`.
- Make sure the layout looks like in production (there should be minimal sizing/spacing changes).
- Visit My Home enabling the experimental layout: `/home/:site?flags=home/experimental-layout`.
- Make sure the layout looks good and there are no styling glitches. 
- Try different sites (prelaunch, setup, established, WP for Teams) and make sure the layout is still consistent.
- Apply D42141-code and sandbox the API.
- Visit My Home enabling the experimental layout for an established site without any Publicize connection.
- Make sure there is a task in the primary location for connecting accounts.
